packages/types/src/interfaces/notify-service.ts

```diff
@@ -1,5 +1,5 @@
 export type NotifyPriority = 'low' | 'medium' | 'high';
 
 export interface INotifyService {
-  notify(message: string, prio?: NotifyPriority): Promise<any>;
+  notify(message: any, subtitle?: string, prio?: NotifyPriority): Promise<any>;
```

Comment:
skapa en NotifyServiceOptions-typ där du stoppar in alla optional parametrar och ta in den istället, så man anropar såhär

```
notify("hejsan", { subtitle: "en subtitle" });
```

Reply:
👍🏻